### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by FredPeal**
start with testing for package

### CODE REVIEW

File: src/Transformers/ADF.php [hunk 1]
```diff
@@ -14,28 +14,76 @@
...
+     * getData.
+     *
+     * @return array
+     */
+    public function getData() : array
+    {
+        return $this->data;
+    }
+
+
```
Comment (kaioken) [inline src/Transformers/ADF.php]: @FredPeal extra space?

**Review by kaioken — CHANGES_REQUESTED** [review]

File: tests/unit/AdfTransformerTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        $data = [
+            'firstname' => $faker->name,
+            'lastname' => $faker->address,
+            'email' => $faker->email,
+            'phone' => $faker->phoneNumber
+        ];
+        $leads = new Leads();
+        $leads->saveOrFail($data);
+        $adfTransformer = new ADF($leads, ['template' => 'dealer-content']);
```
Comment (kaioken) [inline tests/unit/AdfTransformerTest.php]: @FredPeal add a test with multiple models

**Review by kaioken — CHANGES_REQUESTED** [review]

File: src/Communicators/ADF.php [hunk 3]
```diff
@@ -26,7 +27,7 @@ public function send() : bool
     {
         $mail = Di::getDefault()->get('mail');
 
-        $mail->to($company->getEmail())
+        $mail->to($this->company->user->getEmail())
```
Comment (kaioken) [inline src/Communicators/ADF.php:30]: @FredPeal should it be sendnow ?

File: src/Hengen.php [hunk 4]
```diff
@@ -3,22 +3,23 @@
...
+use Canvas\Models\Companies;
 use Hengen\Communicators\ADF as ADFCommunicator;
 use Hengen\Contracts\Interfaces\LeadsInterfaces;
 use Hengen\Contracts\Interfaces\TransformerEngine;
 use Hengen\Transformers\ADF;
 
 class Hengen
 {
-    public static function getTransformer(string $type, LeadsInterfaces $leads, Model ...$args)
+    public static function getTransformer(string $type, LeadsInterfaces $leads, array $options = [], Model ...$args)
```
Comment (kaioken) [inline src/Hengen.php:24]: @FredPeal overall all function documentation

File: src/Transformers/ADF.php [hunk 5]
```diff
@@ -14,28 +14,74 @@
 class ADF implements TransformerEngine
 {
     protected array $data;
+    protected Model $lead;
```
Comment (kaioken) [inline src/Transformers/ADF.php]: @FredPeal shoulnt this be a lead interface or a model interface?

**Review by kaioken — CHANGES_REQUESTED** [review]

**Review by rwhite27 — APPROVED** [review]

**Pull request merged**